### ISSUE TITLE
When the distance between two code blocks is close, there is no way to add blank lines

### ISSUE THREAD

**Issue by LvShuaiChao** [remnoteio/remnote-issues#1136]
Now there is no way to add blank lines in this place
![image](https://user-images.githubusercontent.com/35528109/115950156-2b534480-a50c-11eb-8231-a0565e0234bb.png)

**Comment by hannesfrank** [comment]
The supposed way to create a rem between is by going to the first code block and pressing Esc. This should put the cursor on a new line after the first code block where you can press Enter to create a new rem. The Esc part is bugged at the moment though issue already reported.)

Would this work for you? If yes, we can close this issue and wait for the other being fixed. If no, how would you like it being implemented?

As a workaround I create the rem elsewhere (e.g. focus parent and enter) and move it with Tab and Alt + up/down in place.

**Comment by tfiwm** [comment]
Yes, it is hard to do this via keyboard, therefore I usually use mouse in this situation 

![add_rem_between_codeblocks](https://user-images.githubusercontent.com/24322880/116030188-df89c380-a60f-11eb-8396-12d27f88cff2.gif)

**Comment by hannesfrank** [comment]
Nice catch! Did not even notice that these commands existed! Another case for why we need a clean command architecture to just map any command (here "insert rem below") to a shortcut.
